GiniVision/Classes/Core/GiniScreenAPICoordinator+Camera.swift

```diff
@@ -0,0 +1,292 @@
+                                                 position: .left,
+                                                 target: self)
+        
+        cameraViewController.setupNavigationItem(usingResources: helpButtonResource,
+                                                 selector: #selector(showHelpMenuScreen),
+                                                 position: .right,
+                                                 target: self)
+        
+        if giniConfiguration.fileImportSupportedTypes != .none {
+            documentPickerCoordinator.delegate = self
+            
+            if documentPickerCoordinator.isGalleryPermissionGranted {
+                documentPickerCoordinator.startCaching()
+            }
+            
+            if #available(iOS 11.0, *) {
+                addDropInteraction(forView: cameraViewController.view, with: documentPickerCoordinator)
+            }
+        }
+        
+        return cameraViewController
+    }
+    
+    fileprivate func didCaptureAndValidate(_ document: GiniVisionDocument) {
+        if let didCapture = visionDelegate?.didCapture(document:) {
+            didCapture(document)
+        } else if let didCapture = visionDelegate?.didCapture(_:) {
+            didCapture(document.data)
+        } else {
+            fatalError("GiniVisionDelegate.didCapture(document: GiniVisionDocument) should be implemented")
+        }
+    }
+    
+    private func shouldShowOnBoarding() -> Bool {
+        if giniConfiguration.onboardingShowAtFirstLaunch &&
+            !UserDefaults.standard.bool(forKey: "ginivision.defaults.onboardingShowed") {
+            UserDefaults.standard.set(true, forKey: "ginivision.defaults.onboardingShowed")
+            return true
+        } else if giniConfiguration.onboardingShowAtLaunch {
+            return true
+        }
+        
+        return false
+    }
+    
+    private func showOnboardingScreen() {
+        cameraViewController?.hideCameraOverlay()
+        cameraViewController?.hideCaptureButton()
+        cameraViewController?.hideFileImportTip()
+        
+        let vc = OnboardingContainerViewController { [weak self] in
+            guard let `self` = self else { return }
+            self.cameraViewController?.showCameraOverlay()
+            self.cameraViewController?.showCaptureButton()
+            self.cameraViewController?.showFileImportTip()
+        }
+        
+        let navigationController = UINavigationController(rootViewController: vc)
+        navigationController.applyStyle(withConfiguration: giniConfiguration)
+        navigationController.modalPresentationStyle = .overCurrentContext
+        screenAPINavigationController.present(navigationController, animated: true, completion: nil)
+    }
+    
+    private func showMultipageNewFeatureDialog() {
+        let alertDialog = AlertDialogController(giniConfiguration: giniConfiguration,
+                                                title: "This is the title",
```

Comment:
Are these just sample values? Will they be replaced by real ones? Preferably localized?